bundles/org.openhab.persistence.jdbc/src/main/resources/OH-INF/config/config.xml

```diff
@@ -156,6 +160,11 @@
 			# USE WITH CARE! Deactivate after Renaming is done!
 			#rebuildTableNames=true
 		-->
+		<parameter name="itemsManageTable" type="text">
+			<label>Items Manage Table</label>
+			<description><![CDATA[Items Manage Table <br>(optional, default: "items"). <br>
```

Comment:
No need to set the default tag?

Reply:
You might have a point here. Currently this is defaulted in code the same way as for all other existing configuration parameters.

I think this could be improved though, and I know there are issues when removing a parameter from a configuration file - this will not go back to default in some cases.

Reworking this is probably outside the scope of this pull request, but I can have a look at it.

Reply:
Ok, let's keep that for a future enhancement.